remediation_worker/jobs/azure_security_center_enable_ddos_protection/azure_security_center_enable_ddos_protection.py

```diff
@@ -0,0 +1,135 @@
+import json
+import os
+import sys
+import logging
+
+from azure.mgmt.network import NetworkManagementClient
+from azure.mgmt.network.models import DdosProtectionPlanListResult, SubResource
+from azure.identity import ClientSecretCredential
+from azure.core.paging import ItemPaged
+from typing import List
+
+logging.basicConfig(level=logging.INFO)
+
+def logcall(f, *args, **kwargs):
```

Comment:
Remove this function as this is not being used in the file.
Or, you can make use of this, as in [rds_backup_retention_30_days](https://github.com/vmware-samples/secure-state-remediation-jobs/blob/master/remediation_worker/jobs/rds_backup_retention_30_days/rds_backup_retention_30_days.py#L66)